src/pages/exception/404/locales/zh-CN.ts

```diff
@@ -0,0 +1,4 @@
+export default {
+  'exceptionand404.exception.back': '返回首页',
+  'exceptionand404.description.404': '抱歉，你访问的页面不存在。',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'exceptionand404.description.404' is not sorted alphabetically](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)